src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final ColorSupplier colorSupplier = new ColorSupplier();
+        final int random = new Random().nextInt(100);
+        return new Ball(colorSupplier.getRandomColor(), random);
+    }
```

Comment:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)
read more carefully about magic numbers
also, let's add fields with type `Random` and `ColorSupplier` to our class, 
as you did it for `Random` in `ColorSupplier` class